lib/ruby_indexer/lib/ruby_indexer/declaration_listener.rb

```diff
@@ -751,6 +753,41 @@ def handle_module_operation(node, operation)
       end
     end
 
+    sig { params(node: Prism::CallNode).void }
+    def handle_module_function(node)
+      arguments_node = node.arguments
+      return unless arguments_node
+
+      name_argument = arguments_node.arguments.first
+      return unless name_argument.is_a?(Prism::SymbolNode)
```

Comment:
Question: does Ruby allow you to use a string as the argument to `module_function` or does it only accept a symbol? If it accepts a string, we will want to handle it here as well.